esy/SandboxTools.re

```diff
@@ -1,13 +1,16 @@
 module StringMap = Map.Make(String);
-module StringSet = Set.Make(String);
-module PathSet = Set.Make(Path);
 module ConfigPath = Config.ConfigPath;
 
 let find = (~name, task: Task.t) => {
   let f = (task: Task.t) => task.pkg.name == name;
   Task.DependencyGraph.find(~f, task);
 };
 
+let findScript = (~sandbox: Sandbox.t, argv: list(string)) => {
```

Comment:
Let's move this to `Package.Scripts` as
```
val find : string -> Package.Scripts.t -> Package.Scripts.script option
```
that way we also don't have to call unsafe `List.hd`.